lib/matplotlib/legend.py

```diff
@@ -323,6 +327,7 @@ def __init__(self, parent, handles, labels,
                                  # box, none use rc
                  shadow=None,
                  title=None,  # set a title for the legend
+                 title_prop=None,  # properties for the legend title
```

Comment:
This must be inserted at the end to not break the API, until we make these keyword-only arguments.